extensions/amp-analytics/0.1/test/test-iframe-transport.js

```diff
@@ -117,6 +119,67 @@ describes.realWin('amp-analytics.iframe-transport', {amp: true}, env => {
     expect(env.win.document.getElementsByTagName('IFRAME')).to.have.lengthOf(0);
   });
 
+  it('creates one PerformanceObserver per vendor type', () => {
+    const createLongTaskObserverSpy = sandbox.spy(
```

Comment:
nit: maybe change the name to `createPerformanceObserverSpy`, this is minor, feel free to leave as it is : )

Reply:
Done